Cgranade/0.6 (#408)
MicrosoftDocs/quantum-docs-pr: articles/libraries/standard/licensing.md

---
# Licensing #

The Quantum Development Kit is provided with an extensive collection of open-source functions and operations, licensed under the [MIT license](https://github.com/Microsoft/Quantum/blob/master/LICENSE.txt).
The portions of the standard library that are portable across target machines can be obtained from the [Microsoft/QuantumLibraries](https://github.com/Microsoft/QuantumLibraries) repository on GitHub, along with other libraries such as  and other libraries, such as the [quantum chemistry library](xref:microsoft.quantum.chemistry.concepts.intro).

Microsoft's Quantum Development Kit also provides specialized library functions and operations that are licensed under a [Microsoft Research license](https://github.com/Microsoft/Quantum-NC/blob/master/LICENSE).
These can be obtained from the [Microsoft/Quantum-NC](https://github.com/microsoft/quantum-nc) repository on GitHub.

> There is also an extensive set of samples that explain and illustrate the use of functions and operations from the standard library and other libraries.

Reviewer: ```suggestion
There are also many samples that explain and illustrate the use of functions and operations from the standard library and other libraries.
```
